Street Trees/STTreeMapViewController.swift

```diff
@@ -65,7 +65,7 @@ class STTreeMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
     let spriteView = SKView(frame: CGRect(origin: .zero, size: STSceneSize))
     var treeEmitter = STPeaceEmitter()
     
-    lazy var fetchedResultsController: NSFetchedResultsController = {
+    lazy var fetchedResultsController: NSFetchedResultsController = { () -> <<error type>> in 
```

Comment:
That fixed it, but now I see that FBAnnotationClusteringSwift module is not recognized.